docs/fub_curta.md

```diff
@@ -26,7 +26,7 @@ In addition, job submissions are assigned the appropriate quality of service (QO
 
 Institute-specific pipeline profiles exists for:
 
-- [methylseq](pipeline/methylseq/fub_curta.md)
+- [methylseq](https://github.com/wassimsalam01/configs/blob/master/docs/pipeline/methylseq/fub_curta.md)
```

Comment:
This should point to the main repo, not your fork - is there any reason why you're keeping the docs on your fork?

Reply:
Done!